src/tests/unit/tests/reports/components/report-sections/report-body.test.tsx

```diff
@@ -80,43 +125,24 @@ describe('ReportBody', () => {
             ...detailsProps,
         };
 
-        const wrapper = shallow(<ReportBody {...props} />);
-
-        expect(wrapper.getElement()).toMatchSnapshot();
+        const renderResult = render(<ReportBody {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
+        expect(getMockComponentClassPropsForCall(BodySection).children).toMatchSnapshot();
```

Comment:
Added a comment about this outside of the review

Reply:
https://github.com/microsoft/accessibility-insights-web/pull/7233#discussion_r1513177222